cnxarchive/tests/test_database.py

```diff
@@ -780,8 +780,12 @@ def test_republish_collection(self, cursor):
         collection_ident = cursor.fetchone()[0]
         cursor.connection.commit()
 
-        new_ident = republish_collection(
-            3, collection_ident, testing.fake_plpy)
+        replublished_submittor = "replublished_submittor"
+        replublished_submitlog = "replublished_submitlog"
+
+        new_ident = republish_collection(replublished_submittor,
```

Comment:
This change to the test shows that if you pass in the submitter and the submitlog, it will end up in the modules table.

It does not show the bug that when a module is published, and a collection is republished, that the collection is going to have the module submitter and submitlog instead of the collection submitter and submitlog.  Do you disagree?

Also, would you like to fix the typos?